### PR TITLE
add a table of contents to the remote debuggable targets doc

### PR DESCRIPTION
**Pull request by clarkbw**
This came out of #720

### CODE REVIEW

**Comment by jlongster** [comment]
I like it. Thanks! The failure is only the console test that was fixed.

File: docs/remotely-debuggable-browsers.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
 ## Remotely debuggable browsers
 
+#### Table Of Contents
+
+[Firefox](#firefox)
+  * [MacOS](#macos)
+    * [Firefox](#firefox-release)
+    * [Firefox Beta](#firefox-beta)
+    * [Firefox Developer Edition](#firefox-developer-edition-aurora)
+    * [Firefox Nightly](#firefox-nightly)
```
Comment (jlongster) [inline docs/remotely-debuggable-browsers.md]: Is it worth having all these sections? The only difference between any of them is the name of the app. This makes it looks like there are big differences between all of them to run this. I'd personally prefer to just have a `MacOS` section and use `Firefox.app` by default and then list the different binary names for different versions and make a note that they made need to use one of those.

**Comment by jasonLaster** [comment]
Looks good!

File: docs/remotely-debuggable-browsers.md (hunk 1 above)
Comment (clarkbw) [inline docs/remotely-debuggable-browsers.md]: I agree it is repetitive.  How about I leave two commands for for Firefox and Nightly to make it less repetitive and have notes about DE and Beta.  I think you said once how you like being able to copy the command directly from the doc is nice and I would definitely miss those two versions.  Plus it keeps it similar to the Chrome listing.

Comment (jlongster) [inline docs/remotely-debuggable-browsers.md]: Sounds good to me!

**Comment by jlongster** [comment]
@clarkbw Thanks for updating! Ready to be merged, right?

**Comment by jasonLaster** [comment]
merging with a squash. looks good

**Pull request merged**